packages/dynamic_layouts/example/test/wrap_example_test.dart

```diff
@@ -23,10 +23,29 @@ void main() {
     expect(find.text('Index 3'), findsOneWidget);
     expect(find.text('Index 4'), findsOneWidget);
 
+    // Material 3 changes the expected layout positioning.
+    final bool usesMaterial3 = (app.theme ?? ThemeData.light()).useMaterial3;
+    final Offset offset0 = usesMaterial3
+        ? Offset(0.0, _getExpectedYOffset(91.0))
+        : const Offset(0.0, 103.0);
+    final Offset offset1 = usesMaterial3
+        ? Offset(65.0, _getExpectedYOffset(121.0))
+        : const Offset(66.0, 124.0);
+    final Offset offset3 = usesMaterial3
+        ? Offset(270.0, _getExpectedYOffset(171.0))
+        : const Offset(271.0, 174.0);
+    final Offset offset4 = usesMaterial3
+        ? Offset(380.0, _getExpectedYOffset(221.0))
+        : const Offset(381.0, 224.0);
+
     // See if they are in expected position.
-    expect(tester.getTopLeft(find.text('Index 0')), const Offset(0.0, 103.0));
-    expect(tester.getTopLeft(find.text('Index 1')), const Offset(66.0, 124.0));
-    expect(tester.getTopLeft(find.text('Index 3')), const Offset(271.0, 174.0));
-    expect(tester.getTopLeft(find.text('Index 4')), const Offset(381.0, 224.0));
+    expect(tester.getTopLeft(find.text('Index 0')), offset0);
+    expect(tester.getTopLeft(find.text('Index 1')), offset1);
+    expect(tester.getTopLeft(find.text('Index 3')), offset3);
+    expect(tester.getTopLeft(find.text('Index 4')), offset4);
   });
 }
+
+double _getExpectedYOffset(double nonWeb) {
+  return kIsWeb ? nonWeb - 0.5 : nonWeb;
```

Comment:
Yikes! Is this a "web is rounding wrong" bug?

Reply:
I brought this up to the team chat as well!

Reply:
I vaguely recall we had a special-case for an off-by-one issue in text for Firefox, but not generally. @mdebbar might know more.